DemoSwiftApp/Samples/SamplesForAPI.swift

```diff
@@ -386,6 +408,23 @@ class SamplesForAPI {
         }
         
         print("activated variation: \(String(describing: variationKey))")
+        
+        // [A3] Asynchronous initialization (aync-await)
+        //      1. A datafile is downloaded from the server and the SDK is initialized with the datafile
+        //      2. Polling datafile periodically.
+        //         The cached datafile is used immediately to update the SDK project config.
+        optimizely = OptimizelyClient(sdkKey: "<Your_SDK_Key>",
+                                      periodicDownloadInterval: 60)
+        if #available(iOS 13, *) {
+            Task { [optimizely] in
```

Comment:
Do we need this explicit capturing [optimizely] here?

Reply:
Discussed above